<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RXGroupListDialog</class>
 <widget class="QDialog" name="RXGroupListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit RX Group</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="buddy">
        <cstring>groupListName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="groupListName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QListWidget" name="groupListWidget">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="groupUp">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../shared/resources.qrc">
           <normaloff>:/icons/up.png</normaloff>:/icons/up.png</iconset>
         </property>
         <property name="shortcut">
          <string>Alt+Up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="groupDown">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../shared/resources.qrc">
           <normaloff>:/icons/down.png</normaloff>:/icons/down.png</iconset>
         </property>
         <property name="shortcut">
          <string>Alt+Down</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addContact">
       <property name="text">
        <string>Add Contact</string>
       </property>
       <property name="shortcut">
        <string>Alt++</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remContact">
       <property name="text">
        <string>Remove Contact</string>
       </property>
       <property name="shortcut">
        <string>Alt+-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../shared/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RXGroupListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RXGroupListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
